src/components/ResultItem/_resultItem.scss

```diff
@@ -0,0 +1,84 @@
+@mixin font-size-with-line-height($fontSize, $lineHeightPx) {
```

Comment:
filename should be kebab case (`result-item.scss`)

Reply:
done.